setup.py

```diff
@@ -117,7 +117,7 @@
     "fugashi>=1.0",
     "GitPython<3.1.19",
     "hf-doc-builder>=0.3.0",
-    "huggingface-hub>=0.11.0,<1.0",
+    "huggingface-hub==0.12.0rc0,<1.0",
```

Comment:
Can you just revert that change so we can keep the fixes only?

Reply:
Done :-)